docs/sources/mimir/migration-guide/migrate-from-cortex.md

```diff
@@ -0,0 +1,183 @@
+    | `/<legacy-http-prefix>/api/v1/cardinality/label_values` | `<prometheus-http-prefix>/api/v1/cardinality/label_values` |
+    | `/api/prom/user_stats`                                  | `/api/v1/user_stats`                                       |
+
+  - Distributor endpoints
+
+    | Legacy endpoint              | Current                       |
+    | ---------------------------- | ----------------------------- |
+    | `/<legacy-http-prefix>/push` | `/api/v1/push`                |
+    | `/all_user_stats`            | `/distributor/all_user_stats` |
+    | `/ha-tracker`                | `/distributor/ha_tracker`     |
+
+  - Ingester endpoints
+
+    | Legacy      | Current              |
+    | ----------- | -------------------- |
+    | `/ring`     | `/ingester/ring`     |
+    | `/shutdown` | `/ingester/shutdown` |
+    | `/flush`    | `/ingester/flush`    |
+    | `/push`     | `/ingester/push`     |
+
+  - Ruler endpoints
+
+    | Legacy                                                | Current                                                            |
+    | ----------------------------------------------------- | ------------------------------------------------------------------ |
+    | `/<legacy-http-prefix>/api/v1/rules`                  | `<prometheus-http-prefix>/api/v1/rules`                            |
+    | `/<legacy-http-prefix>/api/v1/alerts`                 | `<prometheus-http-prefix>/api/v1/alerts`                           |
+    | `/<legacy-http-prefix>/rules`                         | `<prometheus-http-prefix>/config/v1/rules`                         |
+    | `/<legacy-http-prefix>/rules/{namespace}`             | `<prometheus-http-prefix>/config/v1/rules/{namespace}`             |
+    | `/<legacy-http-prefix>/rules/{namespace}/{groupName}` | `<prometheus-http-prefix>/config/v1/rules/{namespace}/{groupName}` |
+    | `/<legacy-http-prefix>/rules/{namespace}`             | `<prometheus-http-prefix>/config/v1/rules/{namespace}`             |
+    | `/<legacy-http-prefix>/rules/{namespace}/{groupName}` | `<prometheus-http-prefix>/config/v1/rules/{namespace}/{groupName}` |
+    | `/<legacy-http-prefix>/rules/{namespace}`             | `<prometheus-http-prefix>/config/v1/rules/{namespace}`             |
+    | `/ruler_ring`                                         | `/ruler/ring`                                                      |
+
+  - Alertmanager endpoints
+
+    | Legacy                  | Current                            |
+    | ----------------------- | ---------------------------------- |
+    | `/<legacy-http-prefix>` | `/alertmanager`                    |
+    | `/status`               | `/multitenant_alertmanager/status` |
+
+## Generating configuration for Grafana Mimir
+
+[`mimirtool`]({{< relref "../operators-guide/tools/mimirtool.md" >}}) provides a command for converting Cortex configuration to Mimir configuration that you can use to update both flags and configuration files.
```

Comment:
```suggestion
The [`mimirtool`]({{< relref "../operators-guide/tools/mimirtool.md" >}}) command converts Cortex configuration to Mimir configuration that you can use to update both flags and configuration files.
```

Reply:
how about this?

```suggestion
The [`mimirtool`]({{< relref "../operators-guide/tools/mimirtool.md#config" >}}) `config` command converts Cortex configuration to Mimir configuration. You can use it to update both flags and configuration files.
```

Reply:
Based on https://github.com/grafana/mimir/pull/3929/files#diff-7c7a10d55fd0934f69c75dee10a5c77ebeb80d0764d92529882c930664892560R122, the command looks to be `mimirtool config convert`. In that case, I recommend:

The [`mimirtool config convert`]({{< relref "../operators-guide/tools/mimirtool.md#config" >}}) command converts Cortex configuration to Mimir configuration. You can use it to update both flags and configuration files.